Makefile

```diff
@@ -9,6 +9,9 @@ PKGS = wlroots wayland-server xcb xkbcommon libinput
 CFLAGS += $(foreach p,$(PKGS),$(shell pkg-config --cflags $(p)))
 LDLIBS += $(foreach p,$(PKGS),$(shell pkg-config --libs $(p)))
 
+VERSION=0.7.1beta
```

Comment:
Please don't, I will not manually update this.

Reply:
Let's think of a better versioning mechanism then. Say, a file in the repo root, and an action that changes the versions everywhere.

Reply:
if you can be arsed to do this, sure

Reply:
All you gotta do is edit that file when you make a release. It would still have to be run *before* the release, so that the changed versions end up inside it. Will have to look at how other projects handle it though.

Reply:
You can't create another commit that updates the versions all over the place, because that commit wouldn't correspond to the tagged one.

Reply:
then we'd have to do it all in the release action (forgot we have that now).

Reply:
We don't, @vaxerski manually bumps the tag

Reply:
@vaxerski How would you like this resolved? I guess the most simple way would be to just not include the version in the Man Page, it's not absolutely necessary, just usually included.

But for the long term it would be nice to have some version information automated. I wouldn't mind working to implement this.

Do you know of any other places that currently need a version string?

I would imagine just something simple like `git describe --tags` or whatever could be used. Would not necessarily need a commit, just have it load the version information where necessary during build.

Reply:
idk for now just remove it i guess

Reply:
Okay I've removed it.